http.go

```diff
@@ -49,6 +49,13 @@ type Request struct {
 
 	// To detect scheme changes in redirects
 	schemaUpdate bool
+
+	// Path values are sent as-is without normalization
+	// if this option is set.
+	//
+	// By default path values are normalized, i.e.
+	// extra slashes are removed, special characters are encoded.
+	useOriginalPath bool
```

Comment:
You added this field to `Request` and you set it in `SetUseOriginalPath` but it is never read?

Reply:
Removed